misc fixes (#1180)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/content-block-media.mdx

| ---------------------------------------------------------------------------------------------- | ---------- | -------- | ----------- | ----------------------------------------- |
| Headline                                                                                       | Short copy | Yes      | 1           | XXS (40 / 55)                             |
| Paragraph                                                                                      | Long copy  | No       | 1           | XL (250 / 325)                            |
| [Content group – simple](https://www.ibm.com/standards/carbon/components/content-group-simple) | Component  | Yes      | Less than 4 | –                                         |
| Featured card – medium                                                                         | Component  | No       | 1           | –                                         |
| [Link list](https://www.ibm.com/standards/carbon/components/link-list)                         | Component  | No       | 1           | –                                         |

> You can see the list of all character count standards

Reviewer: ```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

> <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">here</a>.

Reviewer: ```suggestion
```
